src/interfaces/Service.java

```diff
@@ -0,0 +1,97 @@
+package interfaces;
```

Comment:
We shouldn't keep all the interfaces in one package. Interfaces should belong to their respective package that makes sense.
I am waiting for Lee to confirm the base package name and then we can decide package names.
@lccodes FYI

Reply:
done